lib/helper/user_comms.rb

```diff
@@ -0,0 +1,25 @@
+module Emerald
+  class UserCommsHelper
+
+    ERROR_INCORRECT_FILE_TYPE = "Oops! Incorrect file type received.\nExpecting a lisp file e.g. example.lisp\n"
+    ERROR_INITIALISE_WITH_STRING_IO = "Error: Initialize with StringIO objects"
+
+    def initialize(stdout)
+      raise ERROR_INITIALISE_WITH_STRING_IO if stdout.nil?
+      @stdout = stdout if stdout.respond_to?(:puts)
+    end
+
+    def verify_lisp_file(input_file)
+      pattern = /[a-zA-Z0-9\-_]+.lisp/
```

Comment:
I think the definition of a lisp file is a file that can be parsed by your compiler, rather than where the name ends in `.lisp`. After all, I can do this:

```
$ echo "puts 'hello'" >> file.lisp
$ ruby file.lisp
hello